app/uploaders/lighthouse_supporting_evidence_attachment_uploader.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# Files uploaded as part of a form526 submission that will be sent to Lighthouse upon form submission.
+class LighthouseSupportingEvidenceAttachmentUploader < LighthouseDocumentUploaderBase
+  before :store, :log_transaction_start
+  after :store, :log_transaction_complete
```

Comment:
this could all be replaced with the logging wrapper, fyi.

```
wrap_with_logging(
  :store,
      additional_instance_logs: {
        upload_file: %i[size headers]
      }
    )
```
should do the same thing

Reply:
Yeah good catch I had copied this over from the files we duped and missed this was doing basically what 3PL was doing. Fixed in https://github.com/department-of-veterans-affairs/vets-api/pull/13835/commits/44b3e969af1478875b47a970db52bbc6e5e4f7a4